### DISCUSSION TITLE
How to change hitsounds on slider tail?

### DISCUSSION THREAD

**Discussion by Rhythmic-Ocean** [ppy/osu#26143] [Q&A]
Sorry if it's dumb but I don't see an option for it.
![image](https://github.com/ppy/osu/assets/96647127/07c1a751-4938-4ea6-a515-fd32db468df4)

Like in stable you could select the slider end and then on select the hit sound you want, but here, I can't seem to select the slider end at all.

And where is the beatmap upload button btw...

Also, do we have an option to set the hit sound volume for a certain section of the map like in stable (the 'audio' section) in lazer?

**Comment by Theighlin** [comment]
You can't change that yet.
I believe [this](https://github.com/ppy/osu/pull/23443) pr should resolve the issue but the focus now is on gameplay.

**Reply by Rhythmic-Ocean** [reply]
what about uploading beatmap? Where is that button?

**Reply by peppy** [reply]
not yet

**Comment by Eugiv2** [comment]
Not an answer but curious if this is coming soon? 

I'd love to do hitsounding in lazer but if i can't do it correctly with slider head/tails then I see no point in doing it at all and making a poor result. Which really sucks because the changes that have been made are very exciting and I'd love to use them.

**Reply by cihe13375** [reply]
It's already there.

**Reply by Eugiv2** [reply]
is there a new way to select the slider heads/tails in lazer? Clicking on them in the either the grid or timeline doesn't select it for separate hitsounding (as it did it in stable), and adding a hitsound from right-click menu or keyboard shortcuts add to both head and tail of a slider instead of just one side.

**Reply by bdach** [reply]
<img width="459" height="117" alt="image" src="https://github.com/user-attachments/assets/02b54fc1-3e0b-4cf0-9c02-b3d611522e22" />

The thing marked in yellow is what you're supposed to be using for changing tail samples.

**Reply by Eugiv2** [reply]
tysm man. It was probably mentioned in a change log or video I must've missed it.
